.github/workflows/pull_request.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         maven-version: ['3.8.1']
       fail-fast: false
     runs-on: ${{ matrix.os }}
-    name: Maven Build
+    name: Maven - ${{ matrix.os }} - ${{ matrix.java-version }}
```

Comment:
This job seems to have been already broken; see https://github.com/kiegroup/optaplanner/actions/runs/1626493390/workflow